structural_mechanics/use_cases/periodic_bc_example/source/ProjectParameters.json

```diff
@@ -0,0 +1,138 @@
+        "kratos_module" : "KratosMultiphysics",
+        "help"          : "This process sets a vector variable value over a condition",
+        "check"         : "DirectorVectorNonZero direction",
+        "process_name"  : "AssignModulusAndDirectionToConditionsProcess",
+        "Parameters"    : {
+            "mesh_id"         : 0,
+            "model_part_name" : "PointLoad3D_forceOuter",
+            "variable_name"   : "POINT_LOAD",
+            "modulus"         : 0.0,
+            "direction"       : [1.0,0.0,0.0],
+            "interval"        : [0.0,"End"]
+        }
+    },{
+        "python_module" : "apply_periodic_condition_process",
+        "kratos_module" : "KratosMultiphysics",
+        "process_name"  : "ApplyPeriodicBoundaryConditionProcess",
+        "Parameters"    : {
+            "computing_model_part_name"   : "computing_domain",
+            "model_part_name"             : "Structure",
+            "first_model_part_name"       : "GENERIC_horiSurface",
+            "second_model_part_name"      : "GENERIC_vertSurface",
+            "interval"                    : [0.0, 1e30],
+            "variable_names"              : ["DISPLACEMENT"],
+            "transformation_settings"     :{
+                "rotation_settings":{
+                    "center"                      : [0.0,0.0,0.0],
+                    "axis_of_rotation"            : [0.0,0.0,1.0],
+                    "angle_degree"                : 90.0
+
+                }
+            },
+           "search_settings":{
+                    "max_results":100,
+                    "tolerance": 1E-2
+           }
+        }
+    }],
+    "output_configuration"     : {
+        "result_file_configuration" : {
+            "gidpost_flags"       : {
+                "GiDPostMode"           : "GiD_PostAscii",
+                "WriteDeformedMeshFlag" : "WriteDeformed",
+                "WriteConditionsFlag"   : "WriteConditions",
+                "MultiFileFlag"         : "SingleFile"
+            },
+            "file_label"          : "step",
+            "output_control_type" : "step",
+            "output_frequency"    : 1,
+            "body_output"         : true,
+            "node_output"         : false,
+            "skin_output"         : false,
+            "plane_output"        : [],
+            "nodal_results"       : ["DISPLACEMENT","REACTION","VOLUME_ACCELERATION"],
+            "gauss_point_results" : ["VON_MISES_STRESS"]
+        },
+        "point_data_configuration"  : []
+    },
+    "restart_options"          : {
+        "SaveRestart"      : false,
+        "RestartFrequency" : 0,
+        "LoadRestart"      : false,
+        "Restart_Step"     : 0
+    },
+    "constraints_data"         : {
+        "incremental_load"         : false,
+        "incremental_displacement" : false
+    },
+    "material_import_settings" : {
+        "materials_filename" : "StructuralMaterials.json"
+    }
```

Comment:
pls remove `restart_options`, `constraints_data` and `material_import_settings`
it is legacy and not being used 

BTW I recommend updating your GiD-problemtype, it seems to be very old